packages/dev/core/src/Rendering/renderingGroup.ts

```diff
@@ -249,9 +249,14 @@ export class RenderingGroup {
             sortedArray.sort(sortCompareFn);
         }
 
+        const scene = sortedArray[0].getMesh().getScene();

```

Comment:
I think it's possible that `sortedArray.length==0` at that point, so `sortedArray[0].getMesh()` would fail.

Reply:
I don't think so as all the call to it check if the array has a length > 0:
https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Rendering/renderingGroup.ts#L136
https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Rendering/renderingGroup.ts#L143
https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Rendering/renderingGroup.ts#L148
https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Rendering/renderingGroup.ts#L170

(Unless I missed one entry point?)